locale/en/messages.json

```diff
@@ -719,8 +718,6 @@
   "Runs the {0} web interface, as well as a reverse proxy so that administrative interfaces of {1} Service can be accessed from outside the cluster.": "",
   "SAML": "",
   "SAML 2.0": "",
-  "SIZE (GiB)": "",
```

Comment:
There should be new keys now or did I understand these things incorrectly

Reply:
There are no new keys, because "Size (GiB)" already exists as a key.

Reply:
Ah, gotcha 👍